certbot-dns-rackspace/setup.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.25.0.dev0'
```

Comment:
I believe this should be updated to `0.26.0.dev0`. (Sorry for the delay with this review!)